woof-distro/x86_64/slackware64/15.0/DISTRO_PET_REPOS

```diff
@@ -29,16 +29,13 @@ z|https://raw.githubusercontent.com/puppylinux-woof-CE/woof-CE/${WCE_BRANCH}/woo
 #   ex: Packages-puppy-4-official (note, url paths are in the database)
 
 PET_REPOS='
-z|http://distro.ibiblio.org/puppylinux|Packages-puppy-*-official
-z|http://ftp.nluug.nl/ftp/pub/os/Linux/distr/puppylinux|Packages-puppy-*-official
-z|http://ftp.cc.uoc.gr/mirrors/linux/puppylinux|Packages-puppy-*-official
-z|ftp://mirror.aarnet.edu.au/pub/puppylinux|Packages-puppy-*-official
-z|http://distro.ibiblio.org/puppylinux|Packages-puppy-*-official
-z|http://ftp.nluug.nl/ftp/pub/os/Linux/distr/puppylinux|Packages-puppy-*-official
-z|ftp://ftp.vcu.edu/pub/gnu+linux/puppylinux|Packages-puppy-*-official
-z|http://ftp.cc.uoc.gr/mirrors/linux/puppylinux|Packages-puppy-*-official
-z|http://mirror.aarnet.edu.au/pub/puppylinux|Packages-puppy-*-official
-z|http://mirror.internode.on.net/pub/puppylinux|Packages-puppy-*-official
+z|https://distro.ibiblio.org/puppylinux|Packages-puppy-*-official
+z|https://ftp.nluug.nl/ftp/pub/os/Linux/distr/puppylinux|Packages-puppy-*-official
+z|https://ftp.cc.uoc.gr/mirrors/linux/puppylinux|Packages-puppy-*-official
+z|https://distro.ibiblio.org/puppylinux|Packages-puppy-*-official
```

Comment:
Same as line 32?

Reply:
Sorry, I've overlooked a lot of duplicate strings. Fixed.